src/lib/tag-parsers/generate-page-auto-data.js

```diff
@@ -0,0 +1,43 @@
+import parseOgTags from './og-parser';
+import parseJsonLd from './jsonld-parser';
+import { _get } from "../utils";
+
+const generateOgData = () => {
+    const ogTags = parseOgTags();
```

Comment:
I don't see explicit documentation of format that is produced by this parser. What parser is doing is assembling tree structure objects and building some kind of graph. Unfortunately this graph is not standardized so it will be hard to consume on backed. Can we stick to original format  key: value